test/controllers/tip_controller_test.exs

```diff
@@ -1,9 +1,10 @@
 defmodule Healthlocker.TipControllerTest do
   use Healthlocker.ConnCase
+  alias Healthlocker.Post
+  import Healthlocker.Fixtures
 
-  test "lists all entries on index", %{conn: conn} do
+  test "successfully loads tips index", %{conn: conn} do
```

Comment:
My style for these tests:

```
describe "GET index" do
  test "lists all tips" do
    blah blah
  end

  test "returns 200" do
    assert conn.status == 200
  end
end
```

I'm a big fan of `describe` and `context` blocks within tests, it makes them a lot more readable in the tests and the output. Maybe I'll do a lil PR to demonstrate this.